src/core/TabMenu/TabMenu.stories.tsx

```
@@ -0,0 +1,122 @@
+  <div className="p-8 border border-neutral-200 mt-16 rounded-lg">
+    {children}
+  </div>
+);
+
+const tabs = ["Tab 1", "Tab 2", "Tab 3"];
+
+export const Default = Template.bind({});
+Default.args = {
+  tabs: ["Long Tab 1", "Long Long Tab 2", "Tab 3"],
+  contents: [
+    <ExampleContent>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)